src/DI/PhpReflection.php

```diff
@@ -81,6 +87,9 @@ public static function expandClassName($name, \ReflectionClass $rc)
 		if (empty($name)) {
 			throw new Nette\InvalidArgumentException('Class name must not be empty.');
 
+		} elseif (in_array($name, PHP_VERSION_ID >= 70000 ? ['string', 'int', 'float', 'bool', 'array', 'callable'] : ['array', 'callable'], TRUE)) {
```

Comment:
Should I use `strtolower` to `Nette\Utils\Strings::lower`? Because the latter one makes tests fail unless mbstring extension is loaded. How do you add the requirement for the tests to use some extension (travis, running it manually, etc.)?